test/test_convert.py

```diff
@@ -65,7 +65,8 @@ def test_multi_coord_tuple(self):
 
         with self.assertRaises(TypeError):
             convert._build_coords('[1,2],[3,4]')
-            
+    
+    
     def test_polyline_decode(self):
```

Comment:
This test should've different from the beginning.. 

Can you pls change it to smth more related to ors-py? I.e. calculate a route HD to somewhere with higher elevation (not more than 10 km), take the encoded polyline object, insert it into the test (instead of the one that's in there now) and adjust the `self.assertAlmostEqual` parameters? That'd be fantastic:)